examples/linear_map/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _examples-linear-map:
+
+Iteration of a user-defined linear map
+=======================================
+
+This example illustrates the application of a user-defined linear map via a matrix.
+
+Here, the linear map represents an abstract symplectic transformation of the beam in 6D phase space.
+If desired, the user may interpret the matrix as the one-turn map of a storage ring or circular collider.
+
+The (fractional) tunes (Qx, Qy, Qt) of the map are given by (0.139, 0.219, 0.0250).
+We use a 45.6 GeV electron beam that is invariant under the action of the linear map (matched).
+The horizontal and vertical unnormalized emittances are 0.27 nm and 1.0 pm, respectively.
+
+These parameters are based on the single-beam parameters of FCC-ee (Z-mode) appearing here:
+https://twiki.cern.ch/twiki/bin/view/FCC/FCCeeParameters_CDRBaseline-1_0
```

Comment:
I created a backup on the internet archive page, in case this Twiki page/link ever breaks:
```suggestion
These parameters are based on the `single-beam parameters of FCC-ee (Z-mode) <https://twiki.cern.ch/twiki/bin/view/FCC/FCCeeParameters_CDRBaseline-1_0>`__.
(`backup <https://web.archive.org/web/20250000000000*/https://twiki.cern.ch/twiki/bin/view/FCC/FCCeeParameters_CDRBaseline-1_0>`__).
```